lib/wab/impl/configuration.rb

```diff
@@ -19,21 +19,18 @@ def initialize(usage, options)
         
         opts.on('-c', '--config PATH', String, 'Configuration file.') { |c| config_file = c }
         opts.on('-r', '--require LIBRARY', String, 'Require.')        { |r| require r }
-        opts.on('-v', '--verbose', 'Increate verbosity.')             { log_level += 1 }
-        opts.on('-h', '--help', 'Show this display.')                 {
-          puts opts.help
-          Process.exit!(0)
-        }
-        # Start out with an empty @map and add to it with the command line
-        # args processing. The map will be moved sideways are parsing.
-        add_options(opts, options, '')
+        opts.on('-v', '--verbose', 'Increase verbosity.')             { log_level += 1 }
+        opts.on('-h', '--help', 'Show this display.')                 { puts opts.help; Process.exit!(0) }
```

Comment:
The reason I separated into multiple lines is that the Ruby coding guidelines deprecate multiple statements separated by a semicolon in blocks.

Reply:
I did not know that.. but since its only deprecation, we can change it when they actually remove the support..

Reply:
Its the coding guidelines, not the language spec. I thought you were trying to follow them.

Reply:
since we're not strictly adhering to the guidelines in the repo, it can be sufficiently altered to suit adjacent code..

Reply:
I'll remember that for the next time you want me to change something. :-)